sdks/python/apache_beam/io/gcp/pubsub_integration_test.py

```diff
@@ -94,6 +120,23 @@ class PubSubIntegrationTest(unittest.TestCase):
           }),
       ],
   }
+  EXPECTED_OUTPUT_MESSAGES_BYTES = {
+      'TestDirectRunner': [
+          PubsubMessage(b'data001\xab\xac-seen', {'processed': 'IT'}),
+          PubsubMessage(b'data002\xab\xac-seen', {
+              TIMESTAMP_ATTRIBUTE: '2018-07-11T02:02:50.149000Z',
```

Comment:
Thanks, let's combine this scenario with existing scenario.

Reply:
done